app/services/bots/create_service.rb

```diff
@@ -10,11 +10,12 @@ def initialize(user = nil, namespace = nil, bot_type = nil, params = {}) # ruboc
       super(user, params)
       @namespace = namespace
 
+      current_count = @namespace.namespace_bots.count
       user_params = {
-        email: "#{namespace.type.downcase}_#{namespace.puid.downcase}_bot_#{SecureRandom.hex(5)}@iridanext.com",
+        email: "#{namespace.puid}_BOT_#{format('%03d', (current_count + 1))}@iridanext.com",
         user_type: bot_type,
-        first_name: namespace.type.downcase,
-        last_name: 'Bot'
+        first_name: namespace.puid,
+        last_name: "BOT #{format('%03d', (current_count + 1))}"
```

Comment:
**nitpick (blocking):** Can we change this to `Bot 001`, I don't think `BOT` needs to be all upper case in this context.

Reply:
Updated in d55982d7e963f7e8365a2f47f03f3b8060ec9230